app/controllers/api/encryptables_controller.rb

```diff
@@ -115,6 +115,7 @@ def model_serializer
     "#{model_class.name}Serializer".constantize
   end
 
+  # rubocop:disable Metrics/MethodLength
```

Comment:
you could outsource the whole
```
[:cleartext_username, :cleartext_password, :cleartext_token,
:cleartext_pin, :cleartext_email, :folder_id,
{ cleartext_custom_attr: [:label, :value] }]
```
into  a method and just do `permitted_attrs + credential_attrs`

But not sure if that is much more readable but would probably remove the need for the `rubocop:disable` calls

Reply:
I tried it already, works the same, but for me it was less readable, so I removed it and disabled rubocop for this. But I can still outsource it, if it's wanted

Reply:
yea leave it as it is :v: